pkg/cloud/azure/storagebillingparser.go

```diff
@@ -56,17 +60,40 @@ func (asbp *AzureStorageBillingParser) ParseBillingData(start, end time.Time, re
 	}
 
 	for _, blobName := range blobNames {
-		blobBytes, err2 := asbp.DownloadBlob(blobName, client, ctx)
-		if err2 != nil {
-			asbp.ConnectionStatus = cloud.FailedConnection
-			return err2
-		}
-		err2 = asbp.parseCSV(start, end, csv.NewReader(bytes.NewReader(blobBytes)), resultFn)
-		if err2 != nil {
-			asbp.ConnectionStatus = cloud.ParseError
-			return err2
+		if env.IsAzureDownloadBillingDataToDisk() {
```

Comment:
I don't see the need for this if here, if this is truly is an improvement there is no need for YA env variable. Lets just always do it like this.

Reply:
Made the changes! This is now the default functionality.